docs/reference/analysis/specify-analyzer.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[specify-analyzer]]
+=== Specify an analyzer
+
+{es} offers a variety of ways to specify built-in or custom analyzers:
+
+* By `text` field, index, or query
+* For <<analysis-index-search-time,index or search time>>
+
+[TIP]
+.Keep it simple
+====
+The flexibility to specify analyzers at different levels and for different times
+is great... _but only when it's needed_.
+
+In most cases, a simple approach works best: Specify an analyzer for each
+`text` field, as outlined in <<specify-index-field-analyzer>>. No other
+analyzers need to be specified.
```

Comment:
>  No other analyzers need to be specified.

This is not very clear for me.  Sorry, if I misinterpreted this paragraph.

 Analyzers are only specified for text fields, so it is impossible to specify analyzers for any other field types.

Reply:
That sentence was referring to specifying an index analyzer or field-level search analyzer for a field. However, I agree with you: this paragraph is clearer without that sentence. Thanks!